### PR TITLE
Fix make update

### PR DESCRIPTION
**Pull request by JonZeolla**
# Contributor Comments
This fixes `make update` and also attempts to repair the github actions.

## Pull Request Checklist
Thank you for submitting a contribution to `easy_infra`!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:
 - [X] Ensure that `make build` completes successfully
 - [X] Rebase your branch against the latest commit of the target branch, which likely should be `master`
 - [X] If there is an issue associated with your Pull Request, align your PR title with [this documentation](https://help.github.com/en/articles/closing-issues-using-keywords)

### CODE REVIEW

**Comment by JonZeolla** [comment]
Testing cache

**Comment by JonZeolla** [comment]
Cache test 2

**Comment by JonZeolla** [comment]
Cache test 3

File: Makefile [hunk 1]
```diff
@@ -71,21 +71,21 @@ build-aws: generate-functions
...
 push:
-	@docker push --all-tags seiso/easy_infra
+	@docker push --all-tags $(REMOTE_IMAGE_NAME)
 
 .PHONY: update-apt
 update-apt:
 	@echo "Updating the apt package versions..."
 	@for package in $(APT_PACKAGES); do \
-		version=$$(docker run --rm easy_infra:latest /bin/bash -c "apt-get update &>/dev/null && apt-cache policy $${package} | grep '^  Candidate:' | awk -F' ' '{print \$$NF}'"); \
+		version=$$(docker run --rm $(REMOTE_IMAGE_NAME):latest-az /bin/bash -c "apt-get update &>/dev/null && apt-cache policy $${package} | grep '^  Candidate:' | awk -F' ' '{print \$$NF}'"); \
```
Comment (derekseisollc) [inline Makefile:88]: To test locally I had to change REMOTE_IMAGE_NAME to LOCAL_IMAGE_NAME

File: Makefile [hunk 2]
```diff
@@ -120,7 +120,7 @@ update-github:
 .PHONY: update-terraform
 update-terraform:
 	@echo "Updating the terraform version..."
-	@version=$$(docker run --rm easy_infra:latest /bin/bash -c "tfenv list-remote 2>/dev/null | egrep -v '(rc|alpha|beta)' | head -1"); \
+	@version=$$(docker run --rm $(REMOTE_IMAGE_NAME):latest-minimal /bin/bash -c "tfenv list-remote 2>/dev/null | egrep -v '(rc|alpha|beta)' | head -1"); \
```
Comment (derekseisollc) [inline Makefile:123]: To test locally I had to change REMOTE_IMAGE_NAME to LOCAL_IMAGE_NAME

File: Makefile [hunk 3]
```diff
@@ -16,7 +16,7 @@ endif
 
 
 ## Functions
-get_github_latest_version = $$(docker run --rm easy_infra:latest /bin/bash -c "curl -s https://api.github.com/repos/$(1)/releases/latest | jq -r '.tag_name'")
+get_github_latest_version = $$(docker run --rm alpine:3.13 /bin/ash -c "apk add curl jq &>/dev/null && curl -s https://api.github.com/repos/$(1)/releases/latest | jq -r '.tag_name'")
```
Comment (derekseisollc) [inline Makefile:19]: On line 32 Had to remove update-github to test make update locally.

**Review by derekseisollc — APPROVED** [review]
Other than changes made to makefile in order to build locally everything runs as expected.

File: Makefile (hunk 1 above)
Comment (JonZeolla) [reply Makefile:88]: Good point, this PR depends on some other work I never pushed which needs to be done first.

File: Makefile (hunk 3 above)
Comment (JonZeolla) [reply Makefile:19]: That shouldn't be an issue, can you do a `docker pull alpine:3.13 && make update-github` and see if that works?

**Pull request merged**